drivers/SmartThings/matter-thermostat/profiles/air-purifier-hepa-wind-rock.yml

```diff
@@ -0,0 +1,28 @@
+name: air-purifier-hepa-wind-rock
+components:
+- id: main
+  label: Main
+  capabilities:
+  - id: airPurifierFanMode
+    version: 1
+  - id: fanSpeedPercent
+    version: 1
+  - id: windMode
```

Comment:
We should make the order of `windMode` and `fanOscillationMode` consistent in all the profiles that have both. It's different here than in `fan-wind-rock` for example. Unless we have other guidance, I'd say put `fanOscillationMode` before `windMode`

Reply:
Agreed, I will re-comb to make sure everything is consistent